ow_power_system/src/PowerSystemNode.cpp

```diff
@@ -18,6 +18,10 @@ using namespace std_msgs;
 const string FAULT_NAME_HPD           = "high_power_draw";
 const string FAULT_NAME_HPD_ACTIVATE  = "activate_high_power_draw";
 
+// GLOBAL VARS?
+bool CUSTOM_FAULT_ACTIVATED           = false;
+bool HPD_FAULT_ACTIVATED              = false;
```

Comment:
These can be file (static) variables.  Suggest CamelCase which is easier on the eyes.  All caps are for constants.
```suggestion
static bool CustomFaultActivated = false;
static bool HPDFaultActivated      = false;
```

Reply:
I've actually removed these two variables, since they were tied to the whole 'restrict-faults-to-one-or-the-other' feature that has since been removed.